src/config/database.js

```diff
@@ -1,7 +1,10 @@
 const mongoose = require('mongoose');
 require('dotenv').config();
 
-mongoose.connect(process.env.DATABASE_URL, { useNewUrlParser: true });
+mongoose.connect(process.env.DATABASE_URL, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
```

Comment:
Hi @tunedev, can I know why you added `line 6`?

Reply:
I added it to resolve the deprecation warning I got from MongoDB while trying to start the server locally. Attached is a screenshot of the warning. 
<img width="1144" alt="Screenshot 2019-12-28 at 23 10 22" src="https://user-images.githubusercontent.com/33371467/71550038-be877300-29c7-11ea-832b-4850e8b8c419.png">